control-plane/agents/src/bin/ha/node/server.rs

```diff
@@ -454,23 +457,41 @@ fn parse_uri(new_path: &str) -> Result<ParsedUri, SvcError> {
 struct ParsedUri {
     host: String,
     port: u16,
+    transport: String,
```

Comment:
Should be a vector of transports? Or a enum such as { TCP, RDMA, RDMA+TCP } ?

Reply:
> Should be a vector of transports? Or a enum such as { TCP, RDMA, RDMA+TCP } ?

Yes I'll make it a vector of transports. It'll be done when I implement the behaviour change on csi-node to connect rdma and fallback to tcp if initiator is not capable. With this change, the intention is to maintain the current behaviour.